src/main/java/com/example/helloworld/HelloWorldController.java

```diff
@@ -0,0 +1,18 @@
+package com.example.helloworld;
+
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.time.LocalDateTime;
+
+@RestController
+public class HelloWorldController {
+    @RequestMapping("/restHelloWorld")
+    String hello() {
```

Comment:
これでも動きはしますが、public付けておきましょう。
```suggestion
    public String hello() {
```